apps/sms/views/shared/js/contacts.js

```diff
@@ -116,7 +116,7 @@
          *
          */
         return Promise.resolve(
-          filter.filterValue === undefined ? null : []
+          filter.filterValue === undefined ? [] : []
```

Comment:
That means you can change it just to 
```
return Promise.resolve([]);
```
Right? :)

Also there is one more (more important) place where we return null right now: https://github.com/mozilla-b2g/gaia/blob/8a1e4ae522c121c5cacd39b20a5386ec9055db82/apps/sms/views/shared/js/contacts.js#L204